tests/test_wave_01.py

```diff
@@ -116,14 +116,15 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     updated_data = watch_movie(janes_data, MOVIE_TITLE_1)
 
     # Assert
-    assert len(updated_data["watchlist"]) is 0
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watchlist"]) == 0
+    assert len(updated_data["watched"]) == 1
+    assert updated_data["watched"][0] == MOVIE_TITLE_1
```

Comment:
Good! This passes; though if updated_data["watched"] already had pre-populated movies, the new movie would be added to the **end** of the list. Therefore we'd want to use index `-1` or `len(updated_data["watched"])-1` to check that the movie is at the **end** of the list.